### DISCUSSION TITLE
How will we access the localVars hook once MainTemplate gets removed in webpack 6?

### DISCUSSION THREAD

**Discussion by hironarita** [webpack/webpack#14895] [General]
How will we access the localVars hook once MainTemplate gets removed in webpack 6?

**Comment by alexander-akait** [comment]
We don't release webpack v6

**Reply by hironarita** [reply]
I understand it's not released yet but I see this comment in MainTemplate.js

`// TODO webpack 6 remove this class`

**Reply by hironarita** [reply]
I'm working on a webpack plugin that uses the localVars hook from MainTemplate and wondering what will be replacing it.

**Comment by hironarita** [comment]
Is this something I should even be concerned with at this point or cross that bridge once webpack v6 actually gets released?

**Reply by alexander-akait** [reply]
You don't need to worry about it now, many things can be change, it's just a thing for us
